bytecomp/lambda.ml

```diff
@@ -608,14 +616,15 @@ let next_raise_count () =
 let staticfail = Lstaticraise (0,[])
 
 let rec is_guarded = function
```

Comment:
Duplicate comment?

Reply:
Intentionally so. I believe if you end up doing the change I have in mind, you might want to change `is_guarded` to return a `Location.t option` instead of a `bool`. It's just a guess though, we'll see how it goes.